### DISCUSSION TITLE
Img2img - there a way to either impact the 'amount' of variance or the localization of the variation?

### DISCUSSION THREAD

**Discussion by stephantual** [lllyasviel/Fooocus#1599] [Q&A] [answered]
I need a SUPER light touch up of some existing photographs (real ones :)). Even in 'slight' mode, it tries to reinvent objects, face elements, etc. 

So what would be very useful is:
a) a slider from 0 to 10 to let us control it with precision rather 'vary slighly' or 'vary more'.
2) a way to use a mask to impact only part of the image.

I tried and tried but couldn't get it to work - inpainting doesn't 'vary', it repaints.

**Comment by siriume** [comment, marked as answer]
1) Go `Advance` → `Develop` → `Developer Debug Mode` → `Forced Overwrite of Denoising Strength of "Vary"`
Modify it to your value
tips: `Vary (Subtle)` = 0.5, `Vary (Strong)` = 0.85
2) check this #1126 or [this](https://github.com/xhoxye/Fooocus-up_mask/)

**Reply by stephantual** [reply]
Thank you, that's very kind of you
